index.js

```diff
@@ -132,69 +124,56 @@ function onExecute(env) {
     process.env.UNDERTAKER_SETTLE = 'true';
   }
 
+  if (optsErr) {
+    log.error(msgs.error.failToParseCliOpts, optsErr.message);
+    makeHelp(parser).showHelp(console.error);
+    exit(1);
+  }
```

Comment:
This part is needed against an error process of yargs for invalid option.
For example, `gulp -f` causes a yargs error because `-f/—gulpfile` requires an argument. By default, yargs outputs the error message `Not enough arguments following: f`, outputs help messages, and exits the program.
To control this behavior and configure these messages, I added this part and [another part](https://github.com/gulpjs/gulp-cli/pull/161/files#diff-1b0b808a2755d08f27702bc502bda7318339b6c8120d9319eec2b6e5e9e00278L125-R81) in `lib/shared/options/parser.js`.